apps/frontend/app/Header/NavBar.tsx

```diff
@@ -32,30 +34,27 @@ const NavBar = ({
 }: HeaderProps) => {
   const [isMenuOpen, setIsMenuOpen] = useState(false);
   const [activeIcon, setActiveIcon] = useState<string | null>(null);
+  const router = useRouter(); 
 
   const toggleMenu = () => {
     setIsMenuOpen(prev => !prev);
   };
 
   const handleMenuItemClick = (item: string) => {
     console.log(`${item} clicked`);
+    router.push(RutasNavegacion.HomePage)
   };
 
   const handleIconClick = (id: string) => {
     setActiveIcon(id);
     console.log(`Icon clicked: ${id}`);
   };
 
-  const renderIcon = (src: StaticImageData, alt: string, onClick: () => void) => (
-    <Icon src={src} alt={alt} onClick={onClick} />
-  );
-
   return (
     <div className="bg-[#28292E]">
```

Comment:
1. If you are using the div only to apply the background color, you can consider adding this color directly to the header, eliminating the div.
2.  Use semantic HTML.

Reply:
Done